docs/metrics/customizing-the-sdk/README.md

```diff
@@ -251,6 +251,32 @@ by using Views.
 
 See [Program.cs](./Program.cs) for a complete example.
 
+#### Change the ExemplarReservoir
+
+To set the [ExemplarReservoir](#exemplarreservoir) for an instrument, use the
+`MetricStreamConfiguration.ExemplarReservoirFactory` property on the View API:
+
+> [!NOTE]
+> `MetricStreamConfiguration.ExemplarReservoirFactory` is an experimental API only
+  available in pre-release builds. For details see:
+  [OTEL1004](../../diagnostics/experimental-apis/OTEL1004.md).
+
+> [!IMPORTANT]
+> Setting `MetricStreamConfiguration.ExemplarReservoirFactory` alone will NOT
```

Comment:
Could we show this in the example below as well?. A comment can be added about how to change the filter.

Reply:
In the alert there is a link to the exemplar filter section which has that example. We could mirror some of that content here but I think duplicating content (in general) is bad practice. Also how much to duplicate? If we just show calling `SetExemplarFilter` here users might copy that without realizing `OTEL_DOTNET_EXPERIMENTAL_METRICS_EXEMPLAR_FILTER_HISTOGRAMS` is probably a better mechanism 🤣

Reply:
I missed the link :).
The code snippet below looks self-contained as opposed to other snippets above for views. Users might simply copy paste and try running that. May be we can just follow the format used in other view(s) examples?

Reply:
Good call! Updated